acceptance_tests/features/pages/create_message_internal.py

```diff
@@ -24,11 +23,11 @@ def get_ru_details_attributes():
     return ru_details_table_attributes
 
 
-def found_respondent_details():
-    return {'ru_ref': RESPONDENT_DETAILS.get_ru_ref(),
+def found_respondent_details(context):
+    return {'ru_ref': context.short_name,
             'to': 'first_name last_name',
-            'to_uuid': RESPONDENT_DETAILS.get_respondent_id(),
-            'to_ru_id': RESPONDENT_DETAILS.get_ru_id()}
+            'to_uuid': context.respondent_user_name,
```

Comment:
This value should be the respondent's party uuid, not email. I don't think it actually gets used anywhere though, so perhaps we should remove it completely here?

Reply:
deleted "to_ru_id" & "to_uuid" as not used anywhere